 * Remove invalid lines from resolved integration scripts (LP: #1907878)
 -- Heinrich Schuchardt <heinrich.schuchardt@canonical.com>  Sun, 15 May 2022 15:35:51 +0200
--
diff -Nru ifupdown-0.8.36+nmu1ubuntu3/debian/if-down.d/resolved ifupdown-0.8.36+nmu1ubuntu4/debian/if-down.d/resolved
--- ifupdown-0.8.36+nmu1ubuntu3/debian/if-down.d/resolved	2020-07-28 11:55:18.000000000 +0200
+++ ifupdown-0.8.36+nmu1ubuntu4/debian/if-down.d/resolved	2022-05-15 15:28:28.000000000 +0200
@@ -1,6 +1,6 @@
 #!/bin/sh
 #
-# Script fragment to make ifupdown supply DNS infromation to resolved
+# Script fragment to make ifupdown supply DNS information to resolved
 #
 
 case "$ADDRFAM" in
@@ -9,7 +9,6 @@
 esac            
 
 if systemctl is-enabled systemd-resolved > /dev/null 2>&1; then
-    mystatedir statedir ifindex interface
     interface=$IFACE
     if [ ! "$interface" ]; then
         return
diff -Nru ifupdown-0.8.36+nmu1ubuntu3/debian/if-up.d/resolved ifupdown-0.8.36+nmu1ubuntu4/debian/if-up.d/resolved
--- ifupdown-0.8.36+nmu1ubuntu3/debian/if-up.d/resolved	2022-04-17 23:21:44.000000000 +0200
+++ ifupdown-0.8.36+nmu1ubuntu4/debian/if-up.d/resolved	2022-05-15 15:29:07.000000000 +0200
@@ -1,6 +1,6 @@
 #!/bin/sh
 #
-# Script fragment to make ifupdown supply DNS infromation to resolved
+# Script fragment to make ifupdown supply DNS information to resolved
 #
 
 case "$ADDRFAM" in
@@ -9,8 +9,6 @@
 esac            
 
 if systemctl is-enabled systemd-resolved > /dev/null 2>&1; then
-    mystatedir statedir ifindex interface
-
     interface=$IFACE
     if [ ! "$interface" ]; then
         return
